slack_bot/app.py

```diff
@@ -28,21 +29,33 @@ def handler(__event, __context) -> None:
         remove_chosen_users(chosen_users, all_users)
 
 
-def send_message(users: list[str], client: WebClient) -> None:
+def send_message(users: List[str], client: WebClient) -> None:
     response = client.conversations_open(users=users)
     user_name_1 = get_user_name(users[0], client)
     user_name_2 = get_user_name(users[1], client)
     logger.info(f'Send coffee break message to {user_name_1} and {user_name_2}')
 
     client.chat_postMessage(
         channel=response['channel']['id'],
-        text=f'{user_name_1} and {user_name_2}, you were selected for a shared coffee break.\n'
-             'Please schedule a meeting of 15-20 minutes this week.\n\n'
-             'Your coffee bot ☕'
+        text=get_message(user_name_1, user_name_2)
     )
 
 
-def remove_chosen_users(chosen_users: list[str], all_users: list[str]):
+def get_message(user_name_1: str, user_name_2: str) -> str:
+    language = os.environ.get("LANGUAGE", "en")
+
+    if language == "de":
+        return f'{user_name_1} und {user_name_2}, ihr wurdet für einen gemeinsamen Kaffeeklatsch ausgelost.\n' \
+               'Bitte sucht euch für diese Woche einen Zeitslot von 15-20 Minuten.\n\n' \
+               'Euer Kaffeebot ☕'
+
+    # return english as default
+    return f'{user_name_1} and {user_name_2}, you were selected for a shared coffee break.\n' \
+           'Please schedule a meeting of 15-20 minutes this week.\n\n' \
+           'Your coffee bot ☕'
```

Comment:
Ich find's grundsätzlich nicht so schön, die Nachrichten hier in die Funktion zu legen. Man könnte alle Language Templates in eine JSON Datei legen, die dann von der `get_message` Methode gelesen wird. Allerdings wäre das auch irgendwie overkill, weil ich nicht davon ausgehe, das in nächster Zeit mehr Sprachen hinzugefügt werden 😄